docs/README.md

```diff
@@ -0,0 +1,263 @@
+* The granted `roles` will have the permissions defined by Moondog Engine's [`userPermissions` configuration](#userPermissions)
+
+## aws
+
+For typical usage of Moondog Engine on AWS, you should not need to customize the `ebsVolumeTypes`. This configuration will create a `StorageClass` for each of the different volume types in the AWS `region`.
+
+| name | type | description |
+| `----` | ---- | ----------- |
+| `aws.region` | string (optional; default `"us-east-1"`) |  |
+| `aws.ebsVolumeTypes` | list (optional) |  |
+
+## certManager
+
+Configures the [`cert-manager`](https://hub.helm.sh/charts/jetstack/cert-manager) chart to automatically provision TLS certificates from [LetsEncrypt](https://letsencrypt.org/).
+
+| name | type | description |
+| `----` | ---- | ----------- |
+| `certManager.create` | boolean (optional; default `true`) | Whether to create this HelmRelease |
+| `certManager.namespace` | string (optional) | The namespace for the HelmRelease |
+| `certManager.releaseName` | string (optional) | The release name for the HelmRelease |
+| `certManager.replicaCount` | integer (optional; default `1`) | The number of replicas for the deployment |
+| `certManager.email` | string | The support email address reported to LetsEncrypt |
+
+## dex
+
+Configures the [`dex`](https://hub.helm.sh/charts/stable/dex) chart. For typical Moondog Engine usage, the only things you will need to provide are:
+
+* `oauth.clientId`, an arbitrary string that needs to match your Kubernetes API service's `--oidc-client-id` flag.
+* `oauth.clientSecret`, an arbitrary random string which needs to match your Kubernetes API service's `--oidc-client-secret` flag.
+
+Moondog Engine's [`auth` configuration](#auth) will configure your Dex `connectors`, and other apps included in Moondog Engine will configure their own `staticClients` in Dex.
+
+| name | type | description |
+| `----` | ---- | ----------- |
+| `dex.create` | boolean (optional; default `true`) | Whether to create this HelmRelease |
+| `dex.namespace` | string (optional) | The namespace for the HelmRelease |
+| `dex.releaseName` | string (optional) | The release name for the HelmRelease |
+| `dex.port` | integer (optional; default `32000`) |  |
+| `dex.oauth.clientId` |  |  |
+| `dex.oauth.clientSecret` |  |  |
+
+## flux
+
+Configures the [`flux`](https://github.com/fluxcd/flux/tree/master/chart/flux) chart for enabling GitOps. This will allow you to manage cluster resources simply by committing YAML files to a git repository.
+
+| name | type | description |
+| `----` | ---- | ----------- |
+| `flux.create` | boolean (optional; default `true`) | Whether to create this HelmRelease |
+| `flux.namespace` | string (optional) | The namespace for the HelmRelease |
+| `flux.releaseName` | string (optional) | The release name for the HelmRelease |
+| `flux.git.repo` |  |  |
+| `flux.git.branch` |  |  |
+| `flux.auth.key` |  |  |
```

Comment:
I think a few of these options that are blank could use some more explanation (here and in `harbor` for example). Could definitely be a follow-on of course